### PR TITLE
Added french translation

### PR DESCRIPTION
**Pull request by BobyTS**
- Added French translation
- Adding an :shortName alias for servers having a hotelname different from the name of the players.

### CODE REVIEW

**Comment by BobyTS** [comment]
Please excuse me, I have accidentally forgotten to modify the lines commented
" @language en (english) "

File: resources/views/habbo-web-pages/common/box_habbo_way.blade.php [hunk 1]
```diff
@@ -1,2 +1,2 @@
 <h3>{!! trans('common/box_habbo_way.title', ['hotelName' => $chocolatey->hotelName]) !!}</h3>
-<p>{!! trans('common/box_habbo_way.description', ['hotelName' => $chocolatey->hotelName]) !!}</p>
\ No newline at end of file
+<p>{!! trans('common/box_habbo_way.description', ['hotelName' => $chocolatey->hotelName], ['shortName' => $chocolatey->shortName]) !!}</p>
```
Comment (sant0ro) [inline resources/views/habbo-web-pages/common/box_habbo_way.blade.php]: `['hotelName' => $chocolatey->hotelName], ['shortName' => $chocolatey->shortName]` this is wrong

should be 
`['hotelName' => $chocolatey->hotelName, 'shortName' => $chocolatey->shortName]`

File: resources/lang/fr/common/box_account_issues.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * Chocolatey Language Template.
+ *
+ * @template Account Issues Common Box
+ * @uri /common/box_account_issues.en.html
+ * @language en (english)
```
Comment (sant0ro) [inline resources/lang/fr/common/box_account_issues.php]: As said, please change al `@language en (english)` to `@language fr (french)`
And please, change all `.en.html` to `.fr.html`

**Comment by sant0ro** [comment]
@BobyTS you actually can change on this PR the changes that I asked. No need of creating a new one.

**Comment by BobyTS** [comment]
@sant0ro I hope everything is correct :)

**Pull request merged**